imagebuildah/build.go

```diff
@@ -1415,6 +1427,90 @@ func BuildDockerfiles(ctx context.Context, store storage.Store, options BuildOpt
 	return exec.Build(ctx, stages)
 }
 
+// processCopyFrom goes through the Dockerfiles and handles any 'COPY --from' instances
+// prepending a new FROM statement the Dockerfile that do not already have a corresponding
+// FROM command within them.
+func processCopyFrom(dockerfiles []io.ReadCloser) []io.ReadCloser {
+
+	var newDockerfiles []io.ReadCloser
+	// fromMap contains the names of the images seen in a FROM
+	// line in the Dockerfiles.  The boolean value just completes the map object.
+	fromMap := make(map[string]bool)
+	// asMap contains the names of the images seen after a "FROM image AS"
+	// line in the Dockefiles.  The boolean value just completes the map object.
+	asMap := make(map[string]bool)
+	for _, dfile := range dockerfiles {
+		if dfileBinary, err := ioutil.ReadAll(dfile); err == nil {
+			dfileString := fmt.Sprintf("%s", dfileBinary)
+			copyFromContent := strings.Split(dfileString, "COPY --from=")
```

Comment:
should we use a regex here? It seems to not handle Dockerfiles when tabs are used (e.g. `COPY \t --from=myname\n`). The same Dockerfile works with Docker

Reply:
Yet another twist, thought I'd found them all.  Great catch @giuseppe, will see what I can do with it.  I was hoping to avoid a regex, but that might be the answer.

Reply:
I think a good regex for that is `"^\s*COPY\s+--from=([\S]+)\s*$"`